reportportal_client/service.py

```diff
@@ -85,6 +85,25 @@ def _get_messages(data):
     return error_messages
 
 
+def _perform_reliably(request, **kwargs):
+    """Fix for https://github.com/reportportal/client-Python/issues/94
+    Attempt to perform provided HTTP request with kwargs,
```

Comment:
1 blank line required between summary line and description

Reply:
Done